handler.go

```diff
@@ -249,6 +252,13 @@ func (h *TargetHandler) processEvent(ctxt context.Context, msg *cdp.Message) err
 		return nil
 	}
 
+	// Trigger registered event listeners by giving the event / method type
+	if listeners, ok := h.listeners[msg.Method]; ok {
+		for _, listener := range listeners {
+			listener <- msg.Method
```

Comment:
Shouldn't the listeners receive the `ev` event instead? I could imagine just method name without any payload may not be useful.